templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,13 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 42**
+
+* [`PATCH /settings/notifications`](/api/update-notification-settings):
+  Add `enable_notification_on_unsubscribe_stream` setting . This
```

Comment:
Please fix the extra space before the `.` here.  More generally, all of our documentation is highly visible to users and represents the project, so it's important to copy-edit carefully to make sure you're proud of the text and it matches the surrounding style.

Reply:
Oh Sorry I would fix this.